iOS/DuckDuckGo/AppServices/DBPService.swift

```diff
@@ -31,8 +32,9 @@ final class DBPService: NSObject {
                                                                           runTypeProvider: appDependencies.dbpSettings,
                                                                           isAuthV2Enabled: appDependencies.isAuthV2Enabled)
         let authManager = DataBrokerProtectionAuthenticationManager(subscriptionManager: dbpSubscriptionManager)
+        let featureFlagger = DBPFeatureFlagger(appDependencies: appDependencies)
         self.dbpIOSManager = DataBrokerProtectionIOSManagerProvider.iOSManager(authenticationManager: authManager,
-                                                                          privacyConfigurationManager: ContentBlocking.shared.privacyConfigurationManager)
+                                                                               privacyConfigurationManager: ContentBlocking.shared.privacyConfigurationManager, featureFlagger: featureFlagger)
```

Comment:
Super minor, but featureFlagger should be on the next line

Reply:
Good catch!